packages/wallet/src/cip30.ts

```diff
@@ -41,7 +41,7 @@ export type SignDataCallbackParams = {
 
 export type SignTxCallbackParams = {
   type: Cip30ConfirmationCallbackType.SignTx;
-  data: Cardano.NewTxBodyAlonzo;
+  data: Cardano.NewTxAlonzo;
```

Comment:
@mkazlauskas should this be `NewTxAlonzo` or `TxAlonzo`?

Reply:
NewTxAlonzo :+1: